pysos.py

```diff
@@ -126,7 +127,7 @@ def parseValue(line):
     return value
 
 
-class Dict(dict):
+class Dict(collections.abc.MutableMapping):
```

Comment:
It shouldn't break anything. When you used dict, some methods from it did not worked with the persisted data. Now If something is not implemented, it will cause an error. I added a lot of tests to ensure that everything is implemented and works well.